book-club-ui/src/views/BookView.vue

```diff
@@ -78,7 +97,7 @@ export default class BookView extends Vue {
       </div>
       <div class="buttons">
         <button class="book-reading-button" title="Mark as reading"><i class="fa fa-book-open"></i></button>
-        <button class="book-finished-button" title="Mark as finished"><i class="fa fa-square-check"></i></button>
+        <button class="book-finished-button" title="Mark as finished" @click="markBookAsReadForUser"><i class="fa fa-square-check"></i></button>
```

Comment:
Can you put parentheses at the end of this so it's clear it's a function? 🙂

Reply:
Have pushed out an update to fix this. :)